ballerina/client.bal

```diff
@@ -13,3 +16,7154 @@
+        map<Encoding> queryParamEncoding = {"created": {style: DEEPOBJECT, explode: true}, "expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>Retrieves the details of an existing refund.</p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/refunds/[string refund](map<string|string[]> headers = {}, *GetRefundsRefundQueries queries) returns Refund|error {
+        string resourcePath = string `/v1/refunds/${getEncodedUri(refund)}`;
+        map<Encoding> queryParamEncoding = {"expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>Returns a list of Report Runs, with the most recent appearing first.</p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/reporting/report_runs(map<string|string[]> headers = {}, *GetReportingReportRunsQueries queries) returns FinancialReportingFinanceReportRunList|error {
+        string resourcePath = string `/v1/reporting/report_runs`;
+        map<Encoding> queryParamEncoding = {"created": {style: DEEPOBJECT, explode: true}, "expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>Retrieves the details of an existing Report Run.</p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/reporting/report_runs/[string report_run](map<string|string[]> headers = {}, *GetReportingReportRunsReportRunQueries queries) returns Reporting\.report_run|error {
+        string resourcePath = string `/v1/reporting/report_runs/${getEncodedUri(report_run)}`;
+        map<Encoding> queryParamEncoding = {"expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>Returns a full list of Report Types.</p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/reporting/report_types(map<string|string[]> headers = {}, *GetReportingReportTypesQueries queries) returns FinancialReportingFinanceReportTypeList|error {
```

Comment:
It seems all the resource function names are prefixed with `v1`. Probably we can add the same as a suffix to the base url to avoid the repetition? Let's update the `santations.md` accordingly

Reply:
Fixed with following commits:
- [22fc6b330737680905c4aac30dc49d31f25958f9](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/2/commits/22fc6b330737680905c4aac30dc49d31f25958f9)
- [f8940a9e5385a800e8c98b1053f113880bbf1b85](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/2/commits/f8940a9e5385a800e8c98b1053f113880bbf1b85)